lib/node_modules/@stdlib/_tools/changelog/generate/lib/validate.js

```diff
@@ -0,0 +1,87 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isObject = require( '@stdlib/assert/is-plain-object' );
+var hasOwnProp = require( '@stdlib/assert/has-own-property' );
+var isString = require( '@stdlib/assert/is-string' ).isPrimitive;
+var isPlainObject = require( '@stdlib/assert/is-plain-object' );
+var indexOf = require( '@stdlib/utils/index-of' );
```

Comment:
Rather than `indexOf`, use `array/base/assert/contains`.factory. Then at L34, you can do

```
var isFormat = contains( FORMAT_OPTIONS );
```

and update the check below accordingly.

Reply:
Done.